sovware/directorist
# Directory builder/add/widget support in section widget

## PR Type
What kind of change does this PR introduce?


## Description
How to test the changes:

1. Add new widgets in Related Listings widget from the code [snippet](https://gist.github.com/syedgalib/fe0d549bfd206f56a79327cc9c08cfaf). 
2. Go to `Directory Builder -> Single Page Layout -> Contents`.
3. Remove the exiting Related Listing widget if has already.
4. Add the Related Listing widget from widget list again.
5. Check if new widgets are appearing under the Related Listing widget.
6. Click on `Update` to save the new changes.

## Any linked issues
Fixes #

## Checklist
